sci-physics/thepeg/thepeg-2.3.0.ebuild

```diff
@@ -0,0 +1,111 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools elisp-common java-pkg-opt-2
+
+MY_P=ThePEG-${PV}
+
+DESCRIPTION="Toolkit for High Energy Physics Event Generation"
+HOMEPAGE="http://home.thep.lu.se/ThePEG/"
```

Comment:
```suggestion
HOMEPAGE="http://home.thep.lu.se/ThePEG/ https://thepeg.hepforge.org/"
```

Reply:
Thanks, I squashed it in the first commit and also changed the first HOMEPAGE to https.

Reply:
Unfortunately, changing the first link to `https` prevents access due to [a certificate mismatch issue](https://www.ssllabs.com/ssltest/analyze.html?d=home.thep.lu.se).

Reply:
Good that it works now at least, I dropped the `https` for now.